controllers/dashboard_controller.go

```diff
@@ -407,6 +409,7 @@ func (r *GrafanaDashboardReconciler) fetchDashboardJson(dashboard *v1beta1.Grafa
 
 func (r *GrafanaDashboardReconciler) UpdateStatus(ctx context.Context, cr *v1beta1.GrafanaDashboard) error {
```

Comment:
In https://github.com/grafana-operator/grafana-operator/pull/991 we remove UpdateStatus.
Do we need to bring it back. Worth having a quick discussion about it on next meeting?
Ping @pb82 and @weisdd

Reply:
Well, the only reason I removed it was that hash is no longer assigned there, and, as a result, `return r.Client.Status().Update(ctx, cr)` was the only line within `UpdateStatus`. We can reintroduce `UpdateStatus` for sure if there's a need for any other logic there.